lib/Parse/ParseDecl.cpp

```diff
@@ -970,7 +971,22 @@ bool Parser::parseDifferentiableAttributeArguments(
   SyntaxParsingContext ContentContext(
       SyntaxContext, SyntaxKind::DifferentiableAttributeArguments);
 
-  // Parse optional differentiation parameters, starting with the 'wrt:' label.
+  // Parse optional differentiation parameters, starting with
+  // the 'linear' label (optinal).
```

Comment:
The differentiation parameters aren't directly related to the `linear` label, so I'd change the comment here to:
```suggestion
  // Parse 'linear' label (optional).
```
And leave the original comment (below the newly added logic) unchanged.